policyengine_canada/variables/gov/provinces/qc/tax/income/credits/sa/qc_sa_married_one_eligible.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_canada.model_api import *
+
+
+class qc_sa_married_one_eligible(Variable):
+    value_type = bool
+    entity = Household
+    label = "Quebec senior assistance tax credits for family with only one eligible senior"
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
+
+    def formula(household, period, parameters):
+        p = parameters(period).gov.provinces.qc.tax.income.credits.sa
+
+        person = household.members
+
+        age_eligible = person("age", period) >= p.age_eligibility
+        spouse_eligible = person("qc_sa_spouse_eligible", period)
+
+        return age_eligible ^ spouse_eligible
```

Comment:
What does `^` do? Might you mean `&`?

As a household-level variable, this should return a household-level array, rather than person-level. Also please add a unit test to check how this variable works for a household with multiple people.

Same with `qc_sa_married_both_eligible`

Reply:
> What does `^` do? Might you mean `&`?


`^` is XOR (Exclusive OR). For here, it represents that only one senior in the household is eligible (either the spouse or the head of the household, not both).

Reply:
TIL! Thanks! Please add a comment though as some others might not know about that, or that that's the rule.

Reply:
Sure!